config/main.py

```diff
@@ -1515,6 +1515,18 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
             click.echo("Input {} config file(s) separated by comma for multiple files ".format(num_cfg_file))
             return
 
+    #Validate config_db.json file
+    if file_format == 'config_db':
+        if os.path.exists(DEFAULT_CONFIG_DB_FILE):
+            command = [str(SONIC_CFGGEN_PATH), '-j', str(DEFAULT_CONFIG_DB_FILE)]
+            clicommon.run_command(command, display_cmd=True)
+        if  num_cfg_file > 1:
```

Comment:
No need for this condition, `range` will not give any entries if `num_cfg_file` is 1 or less

Reply:
Taken care